js/answer.js

```diff
@@ -0,0 +1,28 @@
+export default function toggleAnswer() {
+  const cards = document.querySelectorAll("[data-js=card]");
+  cards.forEach((card) => {
+    const answerButton = card.querySelector("[data-js=show__answer]");
+    const answer = card.querySelector("[data-js=answer]");
+    answerButton.addEventListener("click", () => {
+      answer.classList.toggle("hidden__answer");
```

Comment:
Please add functionality for toggling the button text, too. Use `classList.contains("hidden__answer")` to check the state of the answer component.